flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/GenericCatalogTable.java

```diff
@@ -41,28 +41,28 @@
 	private final List<String> partitionKeys;
 	// Properties of the table
 	private final Map<String, String> properties;
-	// Comment of the table
-	private String comment = "This is a generic catalog table.";
+	// Description of the table
+	private String description = "This is a generic catalog table.";
```

Comment:
since you're changing this part, I'm wondering if it makes sense to move the comment to properties, as that's what Hive does. Doing this might simply things.

Reply:
I agree. To add on, I also doubt the necessity to have a predefined description and a predefined detailed description.

Nevertheless, I think we'd better leave them to a different PR. I'll revert this part to make this PR focus on what it intends to accomplish

Reply:
Ok. Please revert and create a followup JIRA to track this.

Reply:
reverted https://issues.apache.org/jira/browse/FLINK-12395